homeassistant/components/yale/manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+  "domain": "yale",
+  "name": "Yale",
+  "codeowners": ["@bdraco"],
+  "config_flow": true,
+  "dhcp": [
+    {
+      "hostname": "yale-connect-plus",
+      "macaddress": "00177A*"
+    }
+  ],
+  "documentation": "https://www.home-assistant.io/integrations/yale",
+  "iot_class": "cloud_push",
+  "loggers": ["pubnub", "yalexs"],
```

Comment:
Didn't you remove pubnub? I can remember a PR where you migrated to websockets

Reply:
Good catch

- [x] yale - fixed in 590e5ad4e08
- [x] august - does not apply